maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/RunningAppRouteHandler.swift

```diff
@@ -0,0 +1,32 @@
+import FlyingFox
+import XCTest
+
+class RunningAppRouteHandler: RouteHandler {
+    private static let springboardBundleId = "com.apple.springboard"
+    
+    func handle(request: HTTPRequest) async throws -> HTTPResponse {
+        let decoder = JSONDecoder()
+        
+        guard let requestBody = try? decoder.decode(RunningAppRequest.self, from: request.body) else {
+            throw ServerError.RunningAppRequestSerializeFailure
```

Comment:
Lets not throw errors and respond back with server codes 4xx in these cases. I think we should treat these as client errors